packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -435,6 +438,15 @@ function returnModalCopy(
 	isRenewal = false
 ): ModalCopy {
 	switch ( productType ) {
+		case 'gift purchase':
+			return {
+				title: String( translate( 'You are about to remove your gift from the cart' ) ),
+				description: String(
+					translate(
+						"When you press Continue, we'll remove all gift products in the cart, and your gift will not be given."
```

Comment:
I am not sure what to do about translations here, but the text that shows up before this pull request doesn't quite make sense in the case of subscription gifting anyway, so maybe we'd consider deploying this as-is without waiting for translations.